ledger-core/virtual/execute/execute.go

```diff
@@ -273,13 +273,14 @@ func (s *SMExecute) stepIsolationNegotiation(ctx smachine.ExecutionContext) smac
 
 	negotiatedIsolation, err := negotiateIsolation(s.methodIsolation, s.execution.Isolation)
 	if err != nil {
-		return ctx.Error(throw.W(err, "failed to negotiate", struct {
+		s.prepareExecutionError(throw.W(err, "failed to negotiate", struct {
```

Comment:
Let's make more verbose error message, like "failed to negotiate call isolation params".